Automattic/wp-calypso
# Update plugin slug for Sensei Pro in the marketplace

## Proposed Changes

* Changes the plugin slug in the plugin marketplace's `Supercharging and monetizing your blog` section.

## Testing Instructions
- In local dev, check to make sure the `sensei-pro` slug is used in the `Supercharging and monetizing your blog` preview section.
<img width="1079" alt="Screenshot 2023-03-07 at 6 45 25 pm" src="https://user-images.githubusercontent.com/68693/223520224-b281fa85-1bf8-4dd3-b231-94756ea17aed.png">
- Note: `Browse All` will currently not show Sensei Pro due to a marketplace bug.

## Pre-merge Checklist
